File: Baekjoon/구현/5597.py
# https://www.acmicpc.net/problem/5597

import sys
input = sys.stdin.readline

students = [False] * 31

for _ in range(28):
    students[int(input())] = True

for i in range(1, 31):
    if not students[i]:
        print(i)

File: Baekjoon/구현/5597.cpp
// https://www.acmicpc.net/problem/5597

#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> v(30);
    for (int i = 0; i < 28; i++) {
        int n;
        cin >> n;
        v[n - 1] = 1;
    }
    for (int i = 0; i < 30; i++) {
        if (v[i] == 0) {
            cout << i + 1 << '\n';
        }
    }
    return 0;
}